MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -108,10 +105,11 @@
                              min_track_vertex_sig_to_remove = cms.double(1.5), # default track arbitration
                              remove_one_track_at_a_time = cms.bool(True),
                              max_nm1_refit_dist3 = cms.double(-1),
-                             max_nm1_refit_distz = cms.double(-1), #(0.005), #0.03#0.005 might be too tight so try to relex it
+                             max_nm1_refit_distz = cms.double(-1), #FIXME 0.005 if use_displaced_lepton
                              ignore_lep_in_refit_distz = cms.bool(True), #do not consider dropping tracks at dz refit step if they are leptons w/ pt > 20 GeV. 
```

Comment:
@abbywarden  does this one mean that when `use_displaced_lepton` is on, this value gets automatically adjusted as needed? If so, maybe the FIXME is resolved already? Same question on L112 below!